pyuvdata/uvdata/uvdata.py

```diff
@@ -5315,14 +5315,13 @@ def _apply_w_proj(self, new_w_vals, old_w_vals, select_mask=None):
         new_w_vals: float or ndarray of float
             New w-coordinates for the baselines, in units of meters. Can either be a
             solitary float (helpful for unphasing data, where new_w_vals can be set to
-            0.0) or an array of shape (Nselect,) (which is Nblts if select_mask=None).
+            0.0) or an array of shape (Nblts,).
         old_w_vals: float or ndarray of float
             Old w-coordinates for the baselines, in units of meters. Can either be a
             solitary float (helpful for unphasing data, where new_w_vals can be set to
```

Comment:
The parenthetical here is currently identical to the "new_w_vals" docstring, which is a little confusing. Should this read "helpful for phasing unprojected data where old_w_vals can be set to 0.0"?

Reply:
Ah, good catch -- little too much blind copy/paste on my part. Edited with the above suggestion in 4190b5a0350e04050230984dcc531890e5c7d023